src/build/prefetch.ts

```diff
@@ -81,27 +82,29 @@ export class PrefetchTransform extends Transform {
       return done();
     }
     this.analyzer.analyzeDependencies.then((depsIndex: DepsIndex) => {
-      let fragmentToDeps = new Map(depsIndex.fragmentToDeps);
+      const fragmentToDeps = new Map(depsIndex.fragmentToDeps);
 
       if (this.config.entrypoint && this.config.shell) {
-        let file = this.fileMap.get(this.config.entrypoint);
+        const file = this.fileMap.get(this.config.entrypoint)!;
+        console.assert(file != null);
```

Comment:
I think this only logs, when I think you want to throw.

https://developer.mozilla.org/en-US/docs/Web/API/console/assert